recipe/0485-contentstate-canvas-region/index.md

```diff
@@ -32,7 +32,7 @@ First, we create an Annotation:
 ```json
 {
   "@context": "http://iiif.io/api/presentation/3/context.json",
-  "id": "https://example.org/import/1",
+  "id": "https://iiif.io/api/cookbook/recipe/0485-contentstate-canvas-region/annotation.json",
```

Comment:
We should use the cookbook jekyll tags mentioned in:

https://github.com/IIIF/cookbook-recipes#in-json-files